queue/config.go

```diff
@@ -59,3 +61,7 @@ func (r *Config) Redis(queueConnection string) (dsn string, database int, queue
 
 	return
 }
+
+func (r *Config) Database(queueConnection string) orm.Orm {
+	return facades.Orm().Connection(queueConnection)
+}
```

Comment:
Trying to get the database connection directly through Orm, not sure if it works?

Reply:
> Trying to get the database connection directly through Orm, not sure if it works?

Replace with `OrmFacade`.